Resources/config/orm.xml

```diff
@@ -75,7 +75,6 @@
     </parameters>
 
     <services>
-        <service id="Doctrine\Common\Persistence\ObjectManager" alias="doctrine.orm.entity_manager" public="false" />
```

Comment:
Since Symfony 4.3 we can depreciate a service like this : https://symfony.com/blog/new-in-symfony-4-3-deprecating-service-aliases
Would adding them in 1.x cause a BC for a lower application 4.3 ?
It's could be very useful if we can do that

Reply:
Ah, I didn't know. This could be added to 1.12.x, can you submit a PR?

Reply:
Of course 😃